SharePointFramework/PortfolioWebParts/src/components/ProjectList/types.ts

```diff
@@ -43,10 +45,36 @@ export interface IProjectListProps extends IBaseComponentProps {
    */
   showProjectManager?: boolean
 
+  /**
+   * Show Life Cycle Status
+   */
+  showLifeCycleStatus?: boolean
+
+  /**
+   * Show Service Area
+   */
+  showServiceArea?: boolean
+
+  /**
+   * Show Type
+   */
+  showType?: boolean
+
   /**
    * Columns
    */
   columns?: IColumn[]
+
+  /**
+   * Term Set IDs
+   */
+   termSetIds?: string
```

Comment:
Should be `termSetId`.

Reply:
Could be plural if the customer chooses to use multiple term sets